webapp/src/shared/components/Header/Header.tsx

```diff
@@ -52,21 +53,28 @@ export default function Header({
       )}
     </>
   );
+  const headerTitle = (
+    <Text
+      variant={TextVariant.HEADING}
+      color={TextColor.LIGHT}
+      weight={TextWeight.BOLD}
+    >
+      {children}
+    </Text>
+  );
   return (
     <header className="header">
       <div className="header-navigation">
-        {(navigationUrl && <Link to={navigationUrl}>{lNavFigure}</Link>) || (
-          <button onClick={onClick}>{lNavFigure}</button>
-        )}
+        {(iconNavigationUrl && (
+          <Link to={iconNavigationUrl}>{lNavFigure}</Link>
+        )) || <button onClick={onClick}>{lNavFigure}</button>}
       </div>
       <div className="header-text">
-        <Text
-          variant={TextVariant.HEADING}
-          color={TextColor.LIGHT}
-          weight={TextWeight.BOLD}
-        >
-          {children}
-        </Text>
+        {titleNavigationUrl ? (
+          <Link to={titleNavigationUrl}>{headerTitle}</Link>
```

Comment:
Creo que así:
![Screenshot from 2022-11-02 11-04-10](https://user-images.githubusercontent.com/2774420/199448878-cbb0a0fc-ee33-4f6f-bb28-8a6d1f400a28.png)

sería mejor que así, que es como está:
![Screenshot from 2022-11-02 11-04-39](https://user-images.githubusercontent.com/2774420/199448926-880ab780-9439-4d44-a822-9480153d40b7.png)

Si no, la clickable area es toda la div, incluido márgenes, que no es lo esperado.

Reply:
suggestion: We could also solve this by removing the margin from the `h3`.
Something like this:

<details>

```diff
diff --git a/webapp/src/shared/components/Header/Header.css b/webapp/src/shared/components/Header/Header.css
index 8a3a515..e0a1efb 100644
--- a/webapp/src/shared/components/Header/Header.css
+++ b/webapp/src/shared/components/Header/Header.css
@@ -24,6 +24,7 @@
   justify-content: center;
 }
 
+.header-text a > *,
 .header-text > * {
   margin: 0;
   text-align: center;

```

</details>